src/testRunners/jasmine/__tests__/JasmineFormatter-test.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2016, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @emails oncall+jsinfra
+ */
+'use strict';
+
+jest.autoMockOff();
+
+var jasmine = require('../../../../vendor/jasmine/jasmine-1.3.0').jasmine;
```

Comment:
can you do something like:

`const VENDOR_PATH = path.resolve(__dirname, '../../../../vendor');`

and use that in your require calls? :)